### PR TITLE
Python 3.5 Support

### PR DESCRIPTION
**Pull request by dhilipsiva**
Tests are passing. And it took only minimal changes to port it to python 3.5 and Django 1.8.6

### CODE REVIEW

**Comment by sehmaschine** [comment]
thanks. there's only one problem: travis currently does not support testing with python 3.5 – I'm not sure we should implement these updates until we can test properly. any objections?

**Comment by dhilipsiva** [comment]
No. I dont have any objections. I was just thinking if this got merged, then I can install it directly from pypi rather than installing from GitHub. No probs. I will close this PR. Thanks :)

**Comment by sehmaschine** [comment]
I'm reopening this issue for future reference. Once travis supports testing with 3.5, we'll merge this.

File: grappelli/dashboard/utils.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 """
 
 # PYTHON IMPORTS
-from __future__ import unicode_literals
```
Comment (jieter) [inline grappelli/dashboard/utils.py]: Why are you removing this import?

Comment (dhilipsiva) [inline grappelli/dashboard/utils.py]: Sorry was not intentional. `2to3` tool removed these imports. I will refactor this shortly.

**Comment by dhilipsiva** [comment]
@sehmaschine Sorry. Wrong PR. Will close this.

**Comment by dhilipsiva** [comment]
Dang! sorry for the confuion again. I realize @sehmaschine reopened this Issue. Dang! Sincere apologies.

**Comment by fdintino** [comment]
These all seem like minor formatting changes that 2to3 performs but that have nothing to do with python 3.5 specifically. Moreover, the unit tests currently pass on python 3.5. So what is this for?

**Comment by dhilipsiva** [comment]
@fdintino Its been a while I dont dont exactly remember the Error. I got an error while trying to port my code to 3.5. Maybe that edge case is not covered in test cases and thats why the passed? I am really not sure. I will close this Issue for now. I am maintaining a py3.5 port elsewhere.
